### DISCUSSION TITLE
The "next export" command has been removed in favor of "output: export" in next.config.js for vercal

### DISCUSSION THREAD

**Discussion by omarmok** [vercel/next.js#59960] [Help] [answered]
### Summary

hi
i have an error when i npm run dev
![Screenshot 2023-12-27 135413](https://github.com/vercel/next.js/assets/21249199/951cc6cf-9cce-4e80-93b8-334be398ca1e)

next config 

const nextConfig = {
  output: 'export',
  reactStrictMode: true,
  images: {
    domains: ['i.postimg.cc'],
    unoptimized: true,
  },

 
};

module.exports = nextConfig;

**Comment by omarmok** [comment]
i can't build an deploy in vercal

**Reply by ananuness** [reply, marked as answer]
Hii! Have you seen if this optional property works for you? Maybe it's because the generated folder

```js
const nextConfig = {
  output: 'export',
 
  // Optional: Change the output directory `out` -> `dist`
   distDir: 'dist',
}
```
> As you can see it [there](https://nextjs.org/docs/pages/building-your-application/deploying/static-exports#configuration). Also, use `next build` instead of `next export` (if you're using).

**Reply by omarmok** [reply]
yes i have change it it give me same result

**Reply by BirgitPohl** [reply]
I'm not deploying. I just created a newly app with the create-next-app command and got this.

I have a command that runs `next build && next export && npm run lint:build`, because of the error, the last command is not executed. I have to run it manually.
Since this error is stopping any code, of course people can not deploy, so on this subject, I have the same issue as people who want to deploy to Vercel.

**Comment by bensontrent** [comment]
Remove `next export` from your package.json then update your next.config file with the export option as described above. See https://nextjs.org/docs/app/building-your-application/deploying/static-exports

// package.json
``` json
"scripts": {
  "build": "next build"
}
```
